hasweb/pages/site/hgtos.md

```diff
@@ -1,36 +1,38 @@
 
 HASGEEK SHALL NOT BE LIABLE FOR ANY SPECIAL, INCIDENTAL, INDIRECT OR CONSEQUENTIAL DAMAGES OF ANY KIND IN CONNECTION WITH THIS AGREEMENT, EVEN IF HASGEEK HAS BEEN INFORMED IN ADVANCE OF THE POSSIBILITY OF SUCH DAMAGES.
 
-We charge for certain services and provide other services without charge. But whether we do charge or not is a commercial decision and we believe it is important that we reserve the right to change pricing and relating policies if we feel the need to do so. But we will alert you when we are about to effect such changes.
+We charge for certain services and provide other services without charge, this is a commercial decision and we believe it is important that we reserve the right to change pricing and relating policies if we feel the need to do so. But we will alert you when we are about to effect such changes. Refer to our product specific ToS for refund and cancellation policies.
 
-## Acceptable Use
+Your use of this website is subject to the following guidelines:
 
-Your use under these terms must not violate applicable laws, must comply with our content guidelines (which we may vary from time to time) and is subject to deletion or suspension if we receive complaints from third parties. Here are our [Acceptable Use Guidelines](terms)
+## Content
 
-## Communications
+HasGeek reserves the right, but has no obligation, to monitor the materials posted on it's websites. HasGeek shall have the right to remove or edit any Content that in its sole discretion violates, or is alleged to violate, any applicable law or either the spirit or letter of these Terms of Use. Notwithstanding this right of HasGeek, YOU REMAIN SOLELY RESPONSIBLE FOR THE CONTENT OF THE MATERIALS YOU POST ON THE HASGEEK WEBSITES AND IN YOUR PRIVATE MESSAGES.
 
-In consideration for providing these services and/or products, we would like the liberty of sending you newsletters, announcements or other communications from time to time. You of course can choose to opt out of this by clicking on an ‘unsubscribe’ link provided in such communications. You may be required to enter a valid phone number while purchasing a service or product on HasGeek, or while availing gifts from its advertisers. By registering your phone number with us, you consent to be contacted by HasGeek via phone calls and/or SMS notifications, in case of any purchase or delivery related updates. HasGeek will not use your personal information to initiate any promotional phone calls or SMS.
+Your use of any information or materials on this website is entirely at your own risk, for which we shall not be liable. It shall be your own responsibility to ensure that any products, services or information available through this website meet your specific requirements.
```

Comment:
I don't know either, it was already in there. 
I guess what it means that just because we title ourselves as a job board doesn't mean that we are _The_ job board for a user's search requirements. Whether the information provided by us benefits the user is the user's responsibility and not ours. So I guess people can't use us when they are unable to find a job through our job board.